test/library/crypto.go

```diff
@@ -0,0 +1,95 @@
+package library
```

Comment:
The lib-go crypto package should have helpers for most of this stuff.

Reply:
lib-go crypto seems like overkill. These 2 functions (`NewServerCertificate` & `NewCertificateAuthorityCertificate`) do all I need for a test in memory and with minimal input. Eventually I'll move them to a library-go package for e2e tests.